packages/widget/src/index.tsx

```diff
@@ -138,11 +154,21 @@ const renderToStringWithStyles = (element: ReactElement): string => {
   return html;
 };
 
-export const Widget: React.FunctionComponent<Props> = (props) => {
+export const Widget: React.FunctionComponent<Props & {
+  expanded?: boolean;
+  setExpanded?: (val: boolean) => void;
```

Comment:
@RomanistNLX what I'm realizing here is that the public API of this component has three props now `initiallyExpanded`, `expanded` and `setExpanded`. I don't think it's a good idea to put the users through that kind of confusion.